src/redux/store/appStore.js

```diff
@@ -1,6 +1,7 @@
-import { createStore } from 'redux'
+import { createStore, combineReducers } from 'redux'
 import todoAppReducer from '../reducers/appReducer'
 
-const store = createStore(todoAppReducer)
+const rootReducer = combineReducers({ todoAppReducer })
```

Comment:
Please do not use `reducer` in app state. I should be sth like 
`({ todos })` or 
`({ todos: todosReducer })`